flycheck.el

```diff
@@ -8103,6 +8103,24 @@ for more information about the custom directories."
                                                  "--print-config" "."))))
     (eq exitcode 0)))
 
+(defun flycheck-eslint--find-working-directory (_checker)
+  "Look for a working directory to run CHECKER in.
+This will either be the directory that contains `.eslintrc' by supports
+configuration files formats, if no such file is found in the directory
+hierarchy, searches `node_modules' directory to detect root of project."
+  (let* ((regex-opt (concat "\\`\\.eslintrc"
```

Comment:
^^ I meant to say that you could use the *function* regex-opt :)  But it's fine like this (just rename the variable)

Reply:
Ho sorry :D